osu.Game/Screens/OnlinePlay/OnlinePlaySubScreenStack.cs

```diff
@@ -1,27 +1,74 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
-using System.Diagnostics;
+using System;
+using System.Collections.Generic;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
 using osu.Framework.Screens;
+using osu.Game.Beatmaps;
+using osu.Game.Rulesets;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Screens.OnlinePlay.Lounge;
 
 namespace osu.Game.Screens.OnlinePlay
 {
     public partial class OnlinePlaySubScreenStack : OsuScreenStack
     {
+        private OsuScreenDependencies dependencies = null!;
+
+        // Note - these are required to be unbound on disposal.
```

Comment:
I dunno if it's just me but I found this comment a bit cryptic as well, so maybe better to be a little more explicit (if I'm even understanding the intent here correctly?)

```suggestion
        // Note - these bindables must be stored to fields of this component to be correctly unbound on disposal.
```